test-suite/Math/NumberTheory/Powers/CubesTests.hs

```diff
@@ -0,0 +1,75 @@
+-- |
+-- Module:      Math.NumberTheory.Powers.CubesTests
+-- Copyright:   (c) 2016 Andrew Lelechenko
+-- Licence:     MIT
+-- Maintainer:  Andrew Lelechenko <andrew.lelechenko@gmail.com>
+-- Stability:   Provisional
+--
+-- Tests for Math.NumberTheory.Powers.Cubes
+--
+
+{-# OPTIONS_GHC -fno-warn-type-defaults #-}
+
+module Math.NumberTheory.Powers.CubesTests
+  ( testSuite
+  ) where
+
+import Test.Tasty
+import Test.SmallCheck.Series
+
+import Data.Functor.Identity
```

Comment:
this might be me being a bit paranoid, but because you're defining orphan instances for a pretty common type (Identity), might it make sense to newtype it and do GND for the shared stuff? or just define a new one?

likewise, where are Positive and Nonnegative defined?